components/pipeline-service/pipelines-as-code/kustomization.yaml

```diff
@@ -1,6 +1,6 @@
 resources:
 - allow-argocd.yaml
-- https://github.com/openshift-pipelines/pipelines-as-code/releases/download/v0.12.0/release.yaml
+- https://github.com/openshift-pipelines/pipelines-as-code/releases/download/v0.14.0/release.yaml
```

Comment:
Could it be updated to newest one 0.14.1?

Reply:
Done, now it is in v0.14.2